### DISCUSSION TITLE
Create a page that will show us the versions application and wrappid is using.

### DISCUSSION THREAD

**Discussion by irkfaisal** [wrappid/.github#15] [General]
I am creating a page that will show us all the versions of current application and wrappid packages versions with all the details.
Issue link : https://github.com/wrappid/wrappid-app/issues/32

**Comment by samhere17** [comment]
I have tried creating a design on how the page should look like:
https://docs.google.com/presentation/d/1WylglV48o6Z50O7EwcPutOnA_ePZpFj5hgM6Kz3U1m8/edit#slide=id.g1f19dbf7b29_0_0

**Comment by techoneel** [comment]
We've achieved what we have in the design provided by @samhere17  for both app and service
- https://demo.wrappid.dev/about
- https://demoapi.wrappid.dev/

Closing this discussion since the decision is taken and worked on it. Please create issue if you guys suggest any changes.
